MonoGame.Framework/Utilities/PlatformInfo.cs

```diff
@@ -28,8 +28,6 @@ public static MonoGamePlatform MonoGamePlatform
                 return MonoGamePlatform.WebGL;
 #elif WINDOWS && DIRECTX
                 return MonoGamePlatform.Windows;
-#elif WINDOWS_UAP
-                return MonoGamePlatform.WindowsUniversal;
```

Comment:
Note [MonoGamePlatform.cs](https://github.com/MonoGame/MonoGame/blob/develop/MonoGame.Framework/Utilities/MonoGamePlatform.cs) still has the enum - remove that too?